controlplane/kubeadm/internal/etcd/util/set.go

```diff
@@ -0,0 +1,212 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Modified copy of k8s.io/apimachinery/pkg/util/sets/int64.go
```

Comment:
What got modified? Curious why we can't use this out of the box

Reply:
hmm, is it mainly modifying for uint64? If so, can we leverage the `set-gen` code generator instead of a hand-edited copy here?

Reply:
Yes, upstream is Int64.

> If so, can we leverage the set-gen code generator instead of a hand-edited copy here?

Funny you should ask. It didn't work for me, and I filed an issue a while back: https://github.com/kubernetes/code-generator/issues/74. Not sure if that problem was fixed since, though.